### DISCUSSION TITLE
Deta new is not working for me.

### DISCUSSION THREAD

**Discussion by KhaledAchech** [deta/cloud-docs#444] [Help]
Hello Everyone,

I'm trying to deploy a simple api made with FastAPI, I'm following the FastAPI and deta documentation, however I'm unable to go past this stage:
`deta new`
everytime I run this command i get the following error:
Error: failed to create new program: No such project

I've noticed the creation of the .deta folder in my project root directory, but nothing in it.
I'm new to FastAPI and this is my first time using deta as well, any help will be much appreciated, thank you ^^

**Comment by rohanshiva** [comment]
Hello, are you running it in an empty folder?

**Reply by KhaledAchech** [reply]
No, I'm running inside of my project folder. 
Here is a screenshot of the project structure : 
![image](https://user-images.githubusercontent.com/24296171/188200206-8a7cbc02-cb34-4285-8338-fe571bbf377d.png)
also here is a screenshot of the command it's under the mentioned project : 
![image](https://user-images.githubusercontent.com/24296171/188200346-8d00bbff-653c-43c4-9743-798c47098974.png)

**Reply by AyushSehrawat** [reply]
https://docs.deta.sh/docs/cli/commands#deta-new


If you specify the project name, mane sure you created it first from the ui interface (web.deta.sh)

**Reply by rohanshiva** [reply]
Yes, as @jnsougata pointed out, please try removing `.deta` and try running `deta new` again from the root of the project folder.

**Reply by AyushSehrawat** [reply]
Yeah, ig cli should fix this issue. Delete the deta folder when running new command or show a warning before downloading

**Reply by KhaledAchech** [reply]
Yes that fixed it for me, thank you ^^

**Reply by KhaledAchech** [reply]
> https://docs.deta.sh/docs/cli/commands#deta-new
> 
> If you specify the project name, mane sure you created it first from the ui interface (web.deta.sh)

I'm sorry, I tried to mark this as an answer but ended up marking the first comment, how do I mark this reply as an answer ?

**Comment by jnsougata** [comment]
delete `.deta` directory and try `deta new`

**Reply by KhaledAchech** [reply]
Apperantly, I forgot to create the project from the ui interface (web.deta.sh).

**Reply by KhaledAchech** [reply]
Thank you for your answer ^^

**Reply by AyushSehrawat** [reply]
> Apperantly, I forgot to create the project from the ui interface (web.deta.sh).

Yeah thought so , since the error said no such project

**Reply by jnsougata** [reply]
> Apperantly, I forgot to create the project from the ui interface (web.deta.sh).

doesn't matter `default` project always exists

**Reply by KhaledAchech** [reply]
> > Apperantly, I forgot to create the project from the ui interface (web.deta.sh).
> 
> doesn't matter `default` project always exists

I don't know much about the `default` project but creating the project from the ui interface fixed the issue for me.
